arrays-intermediate.js

```diff
@@ -15,11 +18,13 @@ function sumArray(numbers) {
   
   Hint: Use the .find() and .startsWith() methods
 **************************************************************/
-function findFirstStringStartingWithLetter(letter, strings) {
-  //TODO: Add your code here
-}
+// function findFirstStringStartingWithLetter(letter, strings) {
+//   var result = strings.find((value,index) => {
+//      return value.startsWith(letter);})
```

Comment:
Here, you could've added `.toLowerCase()` to the value to compare the letter and the string without any issues, instead of changing the letter below from `h` to `H`.